tests/reference/asr-array_04-b483094.stderr

```diff
@@ -0,0 +1,3 @@
+LCOMPILERS_ASSERT failed: $DIR/src/libasr/../libasr/asr.h
+function down_cast(), line number 40 at 
+f != nullptr
```

Comment:
Can someone help me update the reference tests, please? I attempted installing ubuntu 22.04 and 20.04 in a virtual machine, so I could install `mamba` in it and update the references, but none of the ubuntu installations work for me.